### ISSUE TITLE
Feature request: customize drop shadow properties

### ISSUE THREAD

**Issue by jeremygomez** [EddyVerbruggen/Toast-PhoneGap-Plugin#89]
Unless I'm missing something, it looks like the properties for the toast drop shadow are hardcoded in the native source code. It would be nice to be able to pass in customizable values for the shadow, as the values that are currently used are a little heavy-handed IMHO. There are also certain cases where the toast would look better without one at all.

Thanks for the awesome plugin BTW!

**Comment by EddyVerbruggen** [comment]
Thank you! I think you're referring to iOS mostly, right?

**Comment by jeremygomez** [comment]
Yes. Right now I am only using it for iOS. I will be implementing it for Android soon though as well. Does the Android version have a drop shadow as well?

**Comment by EddyVerbruggen** [comment]
As Toast is an Android concept it's much closer to whatever Android version you're running so mostly not. Adding this for iOS should be relatively easy. In the meanwhile you can [change the native files](https://github.com/EddyVerbruggen/Toast-PhoneGap-Plugin/blob/master/src/ios/Toast%2BUIView.m#L23-L27) if you can.
